appraisal.lic

```diff
@@ -10,25 +10,35 @@ class Appraisal
 
   def initialize
     @equipment_manager = EquipmentManager.new
-    @equipment_manager.wear_equipment_set?('standard')
     @settings = get_settings
+    train_list = get_settings.appraisal_training
```

Comment:
This needs to added to `base` with defaults 
```
appraisal_training:
- zills
- art
```
that should capture the current default behaviour